README.rst

```diff
@@ -25,6 +25,7 @@ Installation
 
 Installation to local environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Assuming that this is run on python3(not Anaconda)
```

Comment:
More exactly, AFAIK it's assuming Python 3.6.  @jckdotim Is it correct?

Could you adjust some orthographic things?

- [Please add a space before the opening parenthesis.](https://english.stackexchange.com/a/241411/25117)
- Please adds a period to the end of the sentence.
- Since Python is a proper noun, please use a capital P for Python.  There should be a space between the name and the version number as well.
- Please insert new lines before and after the paragraph.